cogs/induct.py

```diff
@@ -238,6 +237,21 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
                 reason=f"{ctx.user} used TeX Bot slash-command: \"/induct\"",
             )
 
+        intro_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels,
```

Comment:
Retrive shortcut accessors at the top of the function so that it fails fast:

```python
intro_channel: discord.TextChannel | None = discord.utils.get(
    main_guild.text_channels,
    name="introductions",
)
```